GenderPayGap.WebUI/Views/ManageOrganisations/OrganisationReportsTable.cshtml

```diff
@@ -112,7 +112,7 @@
                 {
                     Html = @<text>
                                 <a loadtest-id="create-report-@viewModel.ReportingYear"
-                                    href="@Url.Action("ReportOverview", "ReportOverview", new { encryptedOrganisationId, reportingYear = viewModel.ReportingYear, canTriggerLateSubmissionWarning = true})">
+                                    href="@Url.Action("ReportFiguresGet", "ReportStarting", new { encryptedOrganisationId, reportingYear = viewModel.ReportingYear})">

```

Comment:
Is is true that this should always link to the starting page? I'm not au fait with the new journey, but I would have expected that if a report has already been started, then this should either go to the point in the journey that they need to continue from, or to the overview page. As it is, if someone's already started a report and they want to come back to it later, this takes them right back to the start!

Reply:
I discussed this and we think it should go to the "review page" I'm thinking I check in the controller if a draft exists and send them to the review page if it does. Does that sound like a reasonable approach or would you suggest doing that condition in the view?

Reply:
That approach sounds good to me!